udata/core/dataset/api.py

```diff
@@ -360,40 +399,38 @@ def get(self):
                                    .paginate(args['page'], args['page_size']))
 
 
-@ns.route('/community_resources/<community_resource:community_resource>/',
-          endpoint='community_resource',
-          doc=common_doc)
-@api.doc(params={
-    'community_resource': 'The community resource unique identifier'})
+@ns.route('/community_resources/<crid:community>/',
+          endpoint='community_resource', doc=common_doc)
+@api.doc(params={'community': 'The community resource unique identifier'})
 class CommunityResourceAPI(SingleObjectAPI, API):
     model = CommunityResource
 
     @api.doc('retrieve_community_resource')
     @api.marshal_with(community_resource_fields)
-    def get(self, community_resource):
+    def get(self, community):
         '''Retrieve a community resource given its identifier'''
-        return community_resource
+        return community
 
     @api.secure
     @api.doc(id='update_community_resource',
              body=community_resource_fields)
     @api.marshal_with(community_resource_fields)
-    def put(self, community_resource):
+    def put(self, community):
         '''Update a given community resource'''
-        form = api.validate(CommunityResourceForm, community_resource)
-        form.populate_obj(community_resource)
-        if not community_resource.organization:
-            community_resource.owner = current_user._get_current_object()
-        community_resource.modified = datetime.now()
-        community_resource.save()
-        return community_resource
+        form = api.validate(CommunityResourceForm, community)
+        form.populate_obj(community)
+        if not community.organization:
+            community.owner = current_user._get_current_object()
```

Comment:
Is it necessary ? I think the `CurrentUserField` and the `PublishAsField` handle this.